main/acle.md

```diff
@@ -8852,6 +8854,8 @@ following:
     on return from the function. That is, the function does not use ZA
     to receive data from callers or to pass data back to callers.
 
+*   Every byte of the function's ZA state is initially zero.
```

Comment:
nit: s/initially/initialized to/ ?

Reply:
I'd prefer to avoid the passive tense.  I reworded things a bit instead — does the new version look better?